File: DentOffice.Model/Racun.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DentOffice.Model
{
    public class Racun
    {
        public int RacunId { get; set; }
        public int? KorisnikId { get; set; }
        public int? PregledId { get; set; }
        public double? UkupnaCijena { get; set; }
        public DateTime? DatumIzdavanjaRacuna { get; set; }
        public bool? IsPlaceno { get; set; }

        public virtual Korisnik Korisnik { get; set; }
        public virtual Pregled Pregled { get; set; }
        public string UkupnaCijenaFormatted => UkupnaCijena?.ToString("0.00") ?? "";
    }
}

File: dentofficemobile/lib/models/racun.dart
// -- racun.dart --
import 'package:json_annotation/json_annotation.dart';

import 'korisnik.dart';
import 'pregled.dart';

part 'racun.g.dart';

@JsonSerializable()
class Racun {
  int racunId;
  int? pregledId;
  int? korisnikId;
  double? ukupnaCijena;
  DateTime? datumIzdavanjaRacuna;
  bool? isPlaceno;
  Korisnik korisnik;
  Pregled pregled;
  String ukupnaCijenaFormatted;

  Racun({required this.racunId,required this.korisnikId,required this.pregledId,required this.ukupnaCijena,required this.datumIzdavanjaRacuna,required this.isPlaceno,required this.korisnik,required this.pregled,required this.ukupnaCijenaFormatted,});

  factory Racun.fromJson(Map<String, dynamic> json) => _$RacunFromJson(json);

  Map<String, dynamic> toJson() => _$RacunToJson(this);
}